src/components/itemPage/Item.jsx

```diff
@@ -89,8 +90,39 @@ class Item extends Component {
     );
   }
 
+  listenEventSource = async () => {
+    let SERVICE = (await conf).services[0];
+    const response = await fetch(SERVICE);
+    var myjson = await response.json();
+    this.update_seq = myjson.update_seq;
+
+    let params = this.props.match.params;
+    this.eventSource = new EventSource(`${SERVICE}/_changes?filter=argos/item&feed=eventsource&since=${this.update_seq}&corpus=${params.corpus}`);
+
+    var self=this;
+    this._fetchItem().then(() => {
+      this.eventSource.onmessage = e => {
+        var data = JSON.parse(e.data);
+        if (self.update_seq === undefined || self.update_seq !== data.seq){
```

Comment:
The first clause is unnecessary (if it is undefined it will be different).

Reply:
Not necessarily, data.seq can be undefined if it can't reach the service.
That's why I added it.

Reply:
> Not necessarily, data.seq can be undefined if it can't reach the service.
That's why I added it.

Sorry, but I really doubt that it will be sufficient to handle the case of a service that does not implement `_changes` and `update_seq`. Test it on steatite to have an idea... but I would be surprised if you didn't have an exception...